docs/user/adding-new-metrics.md

```diff
@@ -70,7 +70,15 @@ The `lifetime` parameter of a metric defines when it will be reset. There are th
 
 ### Naming things
 
-Choose the category and names of metrics to be as specific as possible.
+Metric identifiers (the combination of a metric's category and name) must be unique across all metrics that are sent by an application.
+This includes not only the metrics defined in the app's `metrics.yaml`, but the `metrics.yaml` of any Glean-using library that that application uses, including Glean itself.
```

Comment:
I always find "that that" weird to read. Any change to rephrase this? (I'm ok if not, don't spend too much time on this)